Library/Formula/jsoncpp.rb

```diff
@@ -15,11 +15,17 @@ class Jsoncpp < Formula
 
   option :universal
 
+  needs :cxx11
+
   depends_on "cmake" => :build
 
   def install
+    ENV.cxx11
+    ENV.append "CXXFLAGS", "-std=c++11"
```

Comment:
`ENV.cxx11` should be doing some of this intelligently for you. I'm surprised you need either of those lines below that?